src/canmatrix/dbc.py

```diff
@@ -521,7 +521,8 @@ class FollowUps(object):
                     regexp_raw = re.compile(pattern.encode(dbcImportEncoding))
                     temp = regexp.match(decoded)
                     temp_raw = regexp_raw.match(l)
-                    receiver = list(map(str.strip, temp.group(12).split(',')))
+                    receiverTemp = [str(b) for b in temp.group(12).split(',')]
```

Comment:
```suggestion
                    receiver = [b.strip() for b in temp.group(12).split(',')]
```
(and remote `receiver = ` below)